db/migrate/20170203113625_remove_organisation_reference_from_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrganisationReferenceFromContentItems < ActiveRecord::Migration[5.0]
+  def change
+    remove_reference :content_items, :organisation, foreign_key: true
```

Comment:
I would create a migration to migrate the one-to-many foreign keys to the many-to-many table; if we don't do it, this PR will force all developers to populate all the data again. Same applies to the heroku app and staging.

```ruby
class MigrateManyToManyContentItemsOrganisation < ActiveRecord::Migration[5.0]
  class ContentItem < ActiveRecord::Base
    belongs_to :organisation
    has_and_belongs_to_many :organisations
  end

  def change
  MigrateManyToManyContentItemsOrganisation::ContentItem.find_each do |content_item|
      content_item.organisations << content_item.organisation
    end
  end
end
```

This migration need to be inserted before deleting the foreign key.